easybuild/easyblocks/e/elsi.py

```diff
@@ -0,0 +1,94 @@
+##
+# Copyright 2009-2019 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for ELSI, implemented as an easyblock
+
+@author: Miguel Dias Costa (National University of Singapore)
+"""
+import os
+from easybuild.easyblocks.generic.cmakemake import CMakeMake
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.modules import get_software_root, get_software_version
+
+
+class EB_ELSI(CMakeMake):
+    """Support for building ELSI."""
+
+    @staticmethod
+    def extra_options():
+        """Define custom easyconfig parameters for ELSI."""
+
+        extra_vars = {
+            'enable_pexsi': [False, "Enable PEXSI solver", CUSTOM],
```

Comment:
Any reason not to enable it by default?

Reply:
not that I know of (other than false being the default in ELSI)